config/config.toml

```diff
@@ -40,45 +40,129 @@ DepositSimulation = ["secp", "bls", "deposit_data", "events"]
 
 # Modify constants in Constants.java
 [constants]
+# Minimal preset
+
+
+# Misc
+# ---------------------------------------------------------------
+
+# [customized] Just 8 shards for testing purposes
 SHARD_COUNT = 8
+# [customized] unsecure, but fast
 TARGET_COMMITTEE_SIZE = 4
-MAX_BALANCE_CHURN_QUOTIENT = 32
-MAX_INDICES_PER_SLASHABLE_VOTE = 4096
-MAX_EXIT_DEQUEUES_PER_EPOCH = 4
-SHUFFLE_ROUND_COUNT = 90
+# 2**12 (= 4,096)
+MAX_INDICES_PER_ATTESTATION = 4096
+# 2**2 (= 4)
+MIN_PER_EPOCH_CHURN_LIMIT = 4
+# 2**16 (= 65,536)
+CHURN_LIMIT_QUOTIENT = 65536
+# Normalizes base rewards
+BASE_REWARDS_PER_EPOCH = 5
+# [customized] Faster, but unsecure.
+SHUFFLE_ROUND_COUNT = 10
+
+
+# Deposit contract
+# ---------------------------------------------------------------
+# **TBD**
+DEPOSIT_CONTRACT_ADDRESS = '0x1234567890123456789012345678901234567890'
+# 2**5 (= 32)
 DEPOSIT_CONTRACT_TREE_DEPTH = 32
+
+
+# Gwei values
+# ---------------------------------------------------------------
+# 2**0 * 10**9 (= 1,000,000,000) Gwei
 MIN_DEPOSIT_AMOUNT = 1000000000
-MAX_DEPOSIT_AMOUNT = 32000000000
-FORK_CHOICE_BALANCE_INCREMENT = 1000000000
+# 2**5 * 10**9 (= 32,000,000,000) Gwei
+MAX_EFFECTIVE_BALANCE = 32000000000
+# 2**4 * 10**9 (= 16,000,000,000) Gwei
 EJECTION_BALANCE = 16000000000
-GENESIS_FORK_VERSION = 0
-GENESIS_SLOT = 4294967296
-GENESIS_EPOCH = 536870912
-GENESIS_START_SHARD = 0
+# 2**0 * 10**9 (= 1,000,000,000) Gwei
+EFFECTIVE_BALANCE_INCREMENT = 1000000000
+
+
+# Initial values
+# ---------------------------------------------------------------
+GENESIS_FORK_VERSION = '0x00000000'
```

Comment:
I'm pretty sure `GENESIS_FORK_VERSION` is removed

Reply:
I just copy pasted minimal_constants.yaml file from spec, so let's just keep it in there.